java/Cherepnin_Oleh/src/com/zoolatech/lecture3/tasks/_2/DateValidator.java

```diff
@@ -0,0 +1,33 @@
+package com.zoolatech.lecture3.tasks._2;
+
+public class DateValidator implements SmallerValidator {
```

Comment:
The smaller validators need to accept **`a field name`** during the creation

Reply:
Why is this field needed?

Reply:
The field is simply used for the output like
```
“validating that fieldName is less than 10”
```